release-notes/opensearch-sql.release-notes-1.2.3.0.md

```diff
@@ -0,0 +1,6 @@
+### Version 1.2.3.0 Release Notes
+Compatible with OpenSearch and OpenSearch Dashboards Version 1.2.3
```

Comment:
Supposedly compatible with OpenSearch Dashboards 1.2.x, AFAIK we're not planning a Dashboards release.

Reply:
Updated, thanks!